src/server/cluster/cluster_family.h

```diff
@@ -44,6 +44,10 @@ class ClusterFamily {
 
   size_t MigrationsErrorsCount() const ABSL_LOCKS_EXCLUDED(migration_mu_);
 
+  // Helper function to be used from repltakeover flow. It swaps master and replica
+  // slot config transparently for replica node becoming the new master (where it is called).
+  void ReconcileMasterReplicaTakeoverSlots(bool was_master);
```

Comment:
seems better, no?
seems like now comment is not correct, or I do not understand `was_master` variable

Reply:
way way way better! Yes I need to update the comment